openfisca_web_api/openAPI.yml

```
@@ -30,6 +30,39 @@ tags:
   - name: "Calculations"
   - name: "Documentation"
 paths:
+  /dependencies:
+    post:
+      summary: "Find the dependent input variables of an OpenFisca Variable. This simply looks for constant strings and checks if they are variables in the entity. It will not work if you construct the strings programatically."
```

Add some information on the `int` values returned by the endpoint? Say that we are counting occurrences?
And explicit which strings in`It will not work if you construct the strings programatically.`? 
Suggestion:
```suggestion
      summary: "Count the occurrences of the dependent input variables of an OpenFisca Variable. This simply looks for constant strings and checks if they are variables in the entity. It will not work if you construct the strings of variable names programatically."
```